website/components/BottomNavigation/index.js

```diff
@@ -0,0 +1,62 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import StyledLink from 'components/StyledLink';
+import { Typography } from '@strapi/design-system/Typography';
+import { Flex } from '@strapi/design-system/Flex';
+import { useTheme } from '@strapi/design-system/ThemeProvider';
+import Link from 'next/link';
+
+const Wrapper = styled.div`
+  border-top: ${(props) => `1px solid ${props.theme.colors.neutral200}`};
+`;
+
+const BottomNavigation = ({ next, previous }) => {
+  const theme = useTheme();
+  return (
+    <Wrapper theme={theme}>
```

Comment:
you don't have to pass the theme here. It's directly injected by SCs under the hood via context